packages/openactive-integration-tests/test/features/.eslintrc.json

```diff
@@ -9,8 +9,8 @@
   },
   "rules": {
     "object-curly-newline": ["error", {
-        "ObjectExpression": "always",
-        "ObjectPattern": { "multiline": true },
```

Comment:
This lint rule was making it so that even `{}` or `{ x }` required new lines. I changed it to `minProperties: 3` to match the export declaration. Let me know if there are any problems with this

Reply:
It seems like we should have a consistent rule for all of these. But I've opted out of that for now as that would require making lots of (auto-fixed) changes to the codebase than would not be reasonable for this PR

Reply:
This is great, I was wondering about this! Agreed